src/Presence/kahoot.json

```
@@ -94,5 +94,93 @@
+        "message": "Looking at game summary",
+        "description": "Used when the host is viewing the summary of a game"
+    },
+    "kahoot.login": {
+        "message": "Logging in",
+        "description": "Used when a user is logging in"
+    },
+    "kahoot.createHome": {
+        "message": "Viewing the home page",
+        "description": "Used when the user is viewing the create.kahoot.it home page"
+    },
+    "kahoot.discover": {
+        "message": "Viewing the \"Discover\" page",
+        "description": "Used when the user is viewing the discover page"
+    },
+    "kahoot.searchKahoots": {
+        "message": "Searching for Kahoots:",
+        "description": "Used when the user is searching for Kahoots. The search query will be displayed below."
+    },
+    "kahoot.kahootDetails": {
+        "message": "Viewing details of Kahoot:",
+        "description": "Used when the user is viewing the details of a Kahoot. The name of the Kahoot will be displayed below."
+    },
+    "kahoot.kahootProfile": {
+        "message": "Viewing Kahoot profile:",
+        "description": "Used when the user is viewing the profile of a Kahoot"
+    },
+    "kahoot.myKahoots": {
+        "message": "Viewing their Kahoots",
+        "description": "Used when the user is viewing their own kahoots"
+    },
+    "kahoot.userReports": {
+        "message": "Viewing their game reports",
+        "description": "Used when the user is viewing their own game reports"
+    },
+    "kahoot.myCourses": {
+        "message": "Viewing their courses",
+        "description": "Used when the user is viewing their own courses. A course is a collection of Kahoots and other activities."
+    },
+    "kahoot.editingCourse": {
+        "message": "Editing course:",
+        "description": "Used when the user is editing a course. The name of the course will be displayed below."
+    },
+    "kahoot.viewingCourse": {
+        "message": "Viewing course:",
+        "description": "Used when the user is viewing a course. The name of the course will be displayed below."
+    },
+    "kahoot.editingKahoot": {
+        "message": "Editing Kahoot:",
+        "description": "Used when the user is editing a Kahoot. The name of the Kahoot will be displayed below."
+    },
+    "kahoot.previewingKahoot": {
+        "message": "Previewing Kahoot:",
+        "description": "Used when the user is previewing a Kahoot"
+    },
+    "kahoot.liveCourse": {
+        "message": "Conducting a live course",
+        "description": "Used when the user is conducting a live course. A live course is a collection of activities that users interact with in real time."
+    },
+    "kahoot.liveCourseActivity": {
+        "message": "Activity #{0}: {1}",
+        "description": "Used to display the activity being conducted."
+    },
+    "kahoot.feedback": {
+        "message": "Providing feedback",
+        "description": "Used when a player is providing feedback to the host"
```

feedback about ...? the course?
```suggestion
        "description": "Used when a player is providing feedback about the course to the host"
```